homeassistant/components/nextdns/button.py

```diff
@@ -17,7 +15,7 @@
 
 CLEAR_LOGS_BUTTON = ButtonEntityDescription(
     key="clear_logs",
-    name="{profile_name} Clear Logs",
+    name="Clear Logs",
```

Comment:
Only the first word should be capitalised (applies to all entity descripions)
```suggestion
    name="Clear logs",
```